pico_w/wifi/freertos/httpd/FreeRTOSConfig.h

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
i have no problem with this here (and possibly it is best here), just wasn't sure why/if this is different given that we use a common header elsewhere in the examples

Reply:
All the wifi freertos example seems to duplicate FreeRTOSConfig.h. Maybe I should stop them doing that?